cpython/Makefile

```diff
@@ -92,15 +91,15 @@ $(BUILD)/Makefile: $(BUILD)/.patched
 			  --host=wasm32-unknown-emscripten\
 			  --build=$(shell $(BUILD)/config.guess) \
 			  --prefix=$(INSTALL)  \
+			  --with-build-python=$$(which python) \
```

Comment:
should not it be `which python3.11` instead ?

Reply:
Yeah.